<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserRegDialog</class>
 <widget class="QDialog" name="UserRegDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>手机号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>学号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>班级编号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePass">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
font: 12pt &quot;黑体&quot;;
border-style:outset;
border-radius:3px;
}
QLineEdit:focus{
background-color: #a3d2ec;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="leResult">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>300</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
font: 12pt &quot;黑体&quot;;
border-style:outset;
border-radius:3px;
}
QLineEdit:focus{
background-color: #a3d2ec;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leSchoolName">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
font: 12pt &quot;黑体&quot;;
border-style:outset;
border-radius:3px;
}
QLineEdit:focus{
background-color: #a3d2ec;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxClassNumber">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox::up-button
{
	width:0px;
}
QSpinBox::down-button
{
	width:0px;
}
QSpinBox
{
border:none;
font: 11pt &quot;Microsoft Sans Serif&quot;;
}
</string>
   </property>
   <property name="maximum">
    <number>9999999</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>学校名称：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReg">
   <property name="geometry">
    <rect>
     <x>67</x>
     <y>344</y>
     <width>211</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    border:none;
	font: 12pt &quot;黑体&quot;;
	background-color: rgb(0, 163, 255);
    border-radius:3px;
    color:white;
}
QPushButton:hover{
	background-color: rgb(60, 182, 251);
}</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leQues">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
font: 12pt &quot;黑体&quot;;
border-style:outset;
border-radius:3px;
}
QLineEdit:focus{
background-color: #a3d2ec;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxStuNumber">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox::up-button
{
	width:0px;
}
QSpinBox::down-button
{
	width:0px;
}
QSpinBox
{
border:none;
font: 11pt &quot;Microsoft Sans Serif&quot;;
}
</string>
   </property>
   <property name="maximum">
    <number>9999999</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="leUser">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
font: 12pt &quot;黑体&quot;;
border-style:outset;
border-radius:3px;
}
QLineEdit:focus{
background-color: #a3d2ec;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePhone">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
font: 12pt &quot;黑体&quot;;
border-style:outset;
border-radius:3px;
}
QLineEdit:focus{
background-color: #a3d2ec;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>密保问题：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>密保答案：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
